client/js/views/LoginSignup.vue

```diff
@@ -14,6 +14,7 @@
             :dense="$vuetify.breakpoint.mdAndDown ? true : false"
             hint="Username"
             data-cy="username"
+            autocomplete="username"
```

Comment:
Can't really test locally without a valid SSL cert, but following the spec this should work:
- https://www.chromium.org/developers/design-documents/create-amazing-password-forms/
- https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls%3A-the-autocomplete-attribute